scripts/clean.sh

```diff
@@ -13,3 +13,5 @@ source "$SCRIPT_DIR"/init-env.sh
 ./"$SCRIPT_DIR"/delete-statefulset-pods.sh
 ./"$SCRIPT_DIR"/delete-community-operator.sh
 ./"$SCRIPT_DIR"/delete-litmus-operator.sh
+./"$SCRIPT_DIR"/delete-limit-range.sh
```

Comment:
I guess the "delete" script call should be removed as there is no matching "deploy" for "limit-range".

Reply:
I figured we'd keep it here because _maybe_ we'd still want to clean it up if not around?

Reply:
Let's say that someone applies the YAML but then runs clean, that sort of scenario.

Reply:
Yes, it's also reasonable to keep it. It won't fail if the "limit-range" is not present anyway.